src/test/resources/pages/instructorFeedbackResultsAjaxByQuestionViewForHelperTwo.html

```diff
@@ -315,16 +315,16 @@
             <tbody>
               <tr>
                 <td class="middlealign">
-                  Team 1
+                  Team 1</td></div>'"
```

Comment:
That's right. The browser unsanitizes those strings when displaying to us. If it's not correctly handled, the structure will be messed up (i.e. team name is broken into 2 parts) and we can see it immediately.